src/lib/draw-and-pick.js

```diff
@@ -35,12 +35,26 @@ export function drawLayers({layers, pass}) {
       compositeCount++;
     } else if (layer.props.visible) {
 
+      const pickingSelectedColor = layer.selectedObjectPickingColor();
+      if (pickingSelectedColor) {
+        layer.state.model.updateModuleSettings({
+          pickingValid: true,
```

Comment:
Sure, will copy the docs to deck.gl and update them.
This uniform tells VS version of the module if the current picking color set is valid or not. It was added to fix luma instancing example for cases when mouse position goes out of canvas. In the deck.gl, when currently picked layer changes, we set pickingValid to false on old layer.
We can definitely overload pickingColor (set it to null picking color ([0, 0, 0, 0])) and I am ok with it, but this approach adds clarity to code with setting of an extra uniform, lets discuss this.